web/modules/custom/simplytest_tugboat/src/PreviewConfigGenerator.php

```diff
@@ -193,6 +195,7 @@ private function getSetupCommands(array $parameters) {
       }
     }
 
+    $commands[] = 'echo "memory_limit = 512M" >> /usr/local/etc/php/conf.d/my-php.ini';
```

Comment:
We could get crafty here and do some logic that says "oh, project browser is the main project, let's bump this up"

Reply:
https://www.drupal.org/project/simplytest/issues/3308702#comment-15439698

this says 256M should be OK

Reply:
Okay, go with 256 then ☺️